packages/deploy/package.json

```diff
@@ -23,7 +23,7 @@
   "license": "MIT",
   "dependencies": {
     "@ethersproject/abi": "^5.6.3",
-    "@openzeppelin/defender-base-client": "^1.54.0",
+    "@openzeppelin/defender-base-client": "^1.54.1",
```

Comment:
I noticed in some places we use `^` and in others not. Maybe better to do a fixed versioning?